xml/System.Windows.Media.Animation/Storyboard.xml

```diff
@@ -2895,7 +2895,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method sets the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty> attached property on the specified object. For more information about how storyboard targeting works, see <xref:System.Windows.Media.Animation.Storyboard.TargetProperty>. For more information about how attached properties work, see [Attached Properties Overview](/dotnet/framework/wpf/advanced/attached-properties-overview).  
+ This method sets the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%2A> attached property on the specified object. For more information about how storyboard targeting works, see <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%2A>. For more information about how attached properties work, see [Attached Properties Overview](/dotnet/framework/wpf/advanced/attached-properties-overview).  
```

Comment:
```suggestion

This method sets the <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a%> attached property on the specified object. For more information about how storyboard targeting works, see <xref:System.Windows.Media.Animation.Storyboard.TargetProperty_a>. For more information about how attached properties work, see [Attached Properties Overview](/dotnet/framework/wpf/advanced/attached-properties-overview).  
```

Reply:
%2A is the encoded character of a asterisk. It is a usage of xref links for methods. See details here:
https://docs.microsoft.com/en-us/contribute/how-to-write-links#methods

The previous one was incorrect.

Reply:
I think it's fine to use the asterisk, but for properties, it's not necessary, and since it makes it a little harder to read, I like to remove them.